wbooks/src/redux/commentAction/actions.js

```diff
@@ -0,0 +1,42 @@
+import CommentService from '../../app/services/CommentService';
+
+export const actionNames = {
+  GET_COMMENTS: '@@COMMENT/GET_COMMENTS',
+  GET_COMMENTS_SUCCESFULL: '@@COMMENT/GET_COMMENTS_SUCCESFULL',
+  GET_COMMENTS_FAILURE: '@@COMMENT/GET_COMMENTS_FAILURE',
+  POST_COMMENT: '@@COMMENT/POST_COMMENT',
+  POST_COMMENT_SUCCESFULL: '@COMMENT/POST_COMMENT_SUCCESFULL',
+  POST_COMMENT_FAILURE: '@COMMENT/POST_COMMENT_FAILURE'
+};
+
+const actionCreators = {
+  getComments(bookId) {
+    return async dispatch => {
+      dispatch({ type: actionNames.GET_COMMENTS });
+      const response = await CommentService.getComments(bookId);
+      if (response.ok) {
+        dispatch({ type: actionNames.GET_COMMENTS_SUCCESFULL, payload: { comments: response.data } });
```

Comment:
Is it fine to do
```
dispatch({ type: actionNames.GET_COMMENTS });
```
Since this function name is `getComments`.
But in case of dispatching success or failure actions within the original, we'd prefer to do something like:

```
dispatch(actionCreators.getCommentsSuccess(response.data));
```

So in `actionCreators` we can define:
```
getCommentsSuccess(comments) {
  return { type: actionNames.GET_COMMENTS_SUCCESFULL, payload: { comments: response.data } }
}
```